core/scss/components/combo-box/module.scss

```diff
@@ -1,7 +1,72 @@
-@import '../../tools/functions/core';
+@mixin icon-clear_16() { content: char-from-code("f34a"); }
```

Comment:
These mixins should already be defined in the icon library mixins. Import them from @import '~@momentum-ui/icons/scss/mixins'; instead. If you recreate them here, the char-codes could/will change when new icons are added to the library.

Reply:
got it